sdk/tables/test-resources.json

```diff
@@ -53,7 +62,7 @@
         {
             "condition": "[equals(parameters('endpointType'),'storage')]",
             "type": "Microsoft.Storage/storageAccounts",
-            "apiVersion": "[variables('mgmtApiVersion')]",
+            "apiVersion": "2022-05-01",
```

Comment:
It seems this is the only place other than the output connection string we were using `mgmtApiVersion`. I think we should leave this as it used to be to keep consistency between the two. I can't envision a scenario where we'd use a different value here than what we want to be put in the string itself.